src/main/java/de/fraunhofer/aisec/cpg/frontends/cpp/CXXLanguageFrontend.java

```diff
@@ -418,22 +429,26 @@ public void processAttributes(@NonNull Node node, @NonNull IASTAttributeOwner ow
     return list;
   }
 
-  private Expression handleToken(IASTToken token) {
+  private AnnotationMember handleToken(IASTToken token) {
     String code = new String(token.getTokenCharImage());
 
+    Expression expression;
     switch (token.getTokenType()) {
       case 1:
```

Comment:
Same here for the token type, constants would be better for readability or else we need comments

Reply:
No, there isn't.

Reply:
I have added comments